.eslintrc.json

```diff
@@ -168,7 +174,12 @@
     "jsdoc/check-alignment": 2,
     "jsdoc/check-indentation": 2,
     "jsdoc/check-syntax": 2,
-    "jsdoc/check-tag-names": 2,
+    "jsdoc/check-tag-names": [
+      2,
+      {
+        "definedTags": ["packageDocumentation"]
+      }
+    ],
```

Comment:
You should be able to remove this since we don't use `packageDocumentation` any more.